portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -55,9 +60,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment");
+    Query query = new Query(COMM_TYPE);
     PreparedQuery results = datastore.prepare(query);
-    List<Entity> messages = results.asList(FetchOptions.Builder.withLimit(maxResults));
+    List<Entity> messages = results.asList(FetchOptions.Builder.withLimit(MAX_RESULTS));
```

Comment:
Just curious, is there a reason you are limiting the number of results?

Reply:
I got an error if I called `results.asList()` with no arguments, and `withLimit()` was suggested in the documentation as a possible argument. Also, it seems like Walkthrough 6 involves capping the number of results based on user input, so this will probably be more useful then.

Reply:
asList looks like it requires FetchOptions and has no overloaded methods that take zero arguments, so that's probably why asList() with no arguments returns an error:
https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/PreparedQuery.html#aslist

I did some digging and I think you can use FetchOptions.withDefaults() and it will return all results:
https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/FetchOptions.Builder#withdefaults

In fact, it looks like withLimit is just shorthand for FetchOptions.withDefaults().limit(...);
https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/FetchOptions.Builder#withlimit

It's up to you if you want to keep the results limited to 10, but maybe try out withDefaults to see if that works since you might want it later?

Reply:
Thanks for letting me know! I'll experiment a little with withDefaults().